setup.cfg

```diff
@@ -64,6 +64,10 @@ exclude =
 # `pip install dcqc[PDF]` like:
 # PDF = ReportLab; RXP
 
+# Additional dependencies for compute tests
+all =
```

Comment:
Eventually once this package gets more complex, will we want to split `all` into something like `jsonldtest` ?

Reply:
I considered that, but I also don't want one extra per dependency. Once we have the problem of too many dependencies, we can figure out a way of splitting them up. I think this is relatively a non-issue since most people will be using the container image.